python/lsst/ts/ess/mock/ess_temperature_4ch_c.py

```diff
@@ -0,0 +1,34 @@
+# This file is part of ts_Dome.
```

Comment:
This isn't a mock class. This is class holding temperature 4ch data. Move it out of mock directory please.

Reply:
You are right that it strictly speaking isn't a mock class. The ESS CSC has been added to replace the current implementation using SALPY, which does contain a EssTemperature4ChC class. I added this to be able to implement a common architecture and I need to discuss the EssTemperature4ChC class with the external developer and once that has been done we will probably introduce a better solution. Still, I have moved this class to a new data directory.

Reply:
Also, the code isn't part of ts_Dome but of ts_ess (silly copy and paste mistake) so I have corrected that in all Python files that had ts_Dome.